src/main/java/com/cinema/model/Ticket.java

```diff
@@ -0,0 +1,51 @@
+package com.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private MovieSession movieSession;
+    @ManyToOne
+    private User user;
```

Comment:
not sure whether we need many-to-many here

Reply:
I think we don't because one user can have many tix and one session can have many tix, so many-to-one makes sense to me here.